config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml

```diff
@@ -85,10 +88,11 @@ presubmits:
       path_alias: k8s.io/kubernetes
     spec:
       containers:
-        - image: gcr.io/k8s-testimages/kubekins-e2e:v20200329-0d2a077-experimental
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20200329-0d2a077-master
           command:
             - "runner.sh"
             - "./scripts/ci-conformance.sh"
+            - "--use-ci-artifacts"
```

Comment:
Since the e2e-conformance pull job handles normal conformance, update this job to be able to trigger a conformance job against k8s master for this script instead.